subsys/net/ip/connection.c

```diff
@@ -830,11 +830,24 @@ enum net_verdict net_conn_input(enum net_ip_protocol proto, struct net_pkt *pkt)
 	}
 
 	if (IS_ENABLED(CONFIG_NET_DEBUG_CONN)) {
+		int data_len = -1;
+
+		if (IS_ENABLED(CONFIG_NET_IPV4) &&
+		    net_pkt_family(pkt) == AF_INET) {
+			data_len = NET_IPV4_HDR(pkt)->len[0] * 256 +
+				NET_IPV4_HDR(pkt)->len[1];
```

Comment:
Is alignment here right? (I'd prefer myself that instead of exact alignment, we actually just intended continuation line(s), but alas, that's not how we do it AFAIK).

Reply:
Indentation is just fine in my editor.